File: program/find-area-of-a-cube/find_area_of_a_cube.go
package main

import "fmt"

func main() {
	var num int
	fmt.Print("Enter the side of a cube to find its area = ")
	fmt.Scanln(&num)
	cube := 6 * num * num
	fmt.Println("\nArea of Cube with side", num, "=", cube)
}

File: program/find-area-of-a-cube/FindAreaOfACube.cs
using System;
class MainClass {
    public static void Main (string[] args) {
    Console.WriteLine ("Please enter the side : ");
    double side = Convert.ToDouble(Console.ReadLine());
    double cube = 6 * side * side;
    Console.WriteLine("Area of cube : "+cube);
  }
}
